app/models/immunisation_import.rb

```diff
@@ -124,6 +114,14 @@ def count_column(vaccination_record)
 
   def postprocess_rows!
     StatusUpdater.call(patient: patients)
+    SyncPatientTeamJob.perform_later(
+      VaccinationRecord,
+      vaccination_records.map(&:id)
```

Comment:
```suggestion
      vaccination_records.pluck(:id)
```

I don't think `vaccination_records` will be already fetched here, so this avoids it needing to construct an object in memory for each one just to get the ID.